### PR TITLE
make host_postfix optional when setting up ocm control plane

### PR DESCRIPTION
**Pull request by morvencao**
Signed-off-by: morvencao <lcao@redhat.com>

### CODE REVIEW

File: controlplane/README.md [hunk 1]
```diff
@@ -15,14 +15,13 @@ $ make image
 
 Set environment variables firstly and then deploy controlplane.
 * `HUB_NAME` (optional) is the namespace where the controlplane is deployed in. The default is `ocm-controlplane`.
-* `HOST_POSTFIX` (required) is the rouute domain of the OCP.And the format of API server of controlplane is `ocm-controlplane-${HUB_NAME}.${HOST_POSTFIX}`. 
+* `HOST_POSTFIX` (optional) is the route domain of the OCP. And the format of API server of controlplane is `ocm-controlplane-${HUB_NAME}.${HOST_POSTFIX}`.
```
Comment (clyang82) [inline controlplane/README.md:18]: maybe just remove it since you can get the route domain from the openshift console route.

Comment (morvencao) [reply controlplane/README.md]: removed the `HOST_POSTFIX` item.

**Pull request merged**